proposals/Modules.md

```diff
@@ -0,0 +1,65 @@
+## Modules
+
+Modules will provide the basic organization building block in Serulian.
+
+### Basic Definition
+
+A module is defined as a single Serulian file (ending in `.seru`) that **exports** some set of types
+or functions to be accessible to other modules.
+
+### Using modules
+
+Modules can be **imported** using the `from` or `import` keywords, which support import
+from two different sources: local file system and Git.
+
+Syntax Examples:
+
+```
+// Imports the file "myfavoritemodule.seru" directly, adding its exported members into the local context
+import myfavoritemodule
```

Comment:
Interesting... I wonder how they determine when to do said injection. It is certainly rarely seen.